mmdet3d/models/backbones/resnet.py

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from mmdet.models import BACKBONES
+from mmdet.models.backbones.resnet import ResNet
+
+
+@BACKBONES.register_module()
+class CustomResNet(ResNet):
```

Comment:
Maybe we need to consider rename the backbone for Bev feature extractor, if this backbone is only used in `BEVDet`, we could rename it to `BEVResNet`?

Reply:
Still ugly. Define a new class to wrap all kinds of backbones is more elegance and extensible, just like this:
class BEVBackbone(nn.Module):
      self.__init__():
           delete the input convs

      self.forward():
            forward without input convs
The problem is that the backbones have different structures of input convs....

Reply:
Indeed, different backbones have different structure and layer name of input convs, and different backbones have different `input convs` definition, like DLA backbone and so on.